strings/is_palindrome.py

```diff
@@ -7,8 +7,25 @@ def is_palindrome(s):
     True
     >>> is_palindrome("Hello")
     False
+    >>> is_palindrome("Able was I ere I saw Elba")
+    True
+    >>> is_palindrome("racecar")
+    True
+    >>> is_palindrome("Mr. Owl ate my metal worm?")
+    True
     """
-    return s == s[::-1]
+    # Since Punctuation, capitalization, and spaces are usually ignored while checking Palindrome,
+    # we first remove them from our string.
+    cleaned_string = ""
+    for character in s:
+        if character.isalnum():
+            cleaned_string += character
```

Comment:
Replace 4 lines with 1 using str.join() and a generator expression?

Reply:
Done!